datadog-checks-base/tests/test_prometheus.py

```diff
@@ -93,7 +93,7 @@ def bin_data():
 def text_data():
     # Loading test text data
     f_name = os.path.join(os.path.dirname(__file__), 'fixtures', 'prometheus', 'metrics.txt')
-    with open(f_name, 'rb') as f:
+    with open(f_name, 'r') as f:
```

Comment:
Why did we change this?

Reply:
The file is a text file.  On Windows, that was causing the subsequent length check to come up wrong (\r\n vs. \n).  But it's actually a text file; it shouldn't be `b` (and this makes the tests work on both linux & windows)